sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -230,7 +230,9 @@ object JdbcUtils extends Logging {
       // scalastyle:on
     }
 
-    if (answer == null) throw new SQLException("Unsupported type " + sqlType)
+    if (answer == null) {
+      throw new SQLException("Unsupported type " + JDBCType.valueOf(sqlType).getName)
```

Comment:
Hi, @gatorsmile .
Then, it seems that we need to consider `IllegalArgumentException` from `JDBCType.valueOf` then.
Can we do that here?